assets/constants.ts

```diff
@@ -5891,5 +5891,8 @@ export default {
   ],
 
   "DEV_MODE": true,
-  "EMULATOR_MODE": true
+  "EMULATOR_MODE": true,
+  "FILENAME_PREFIX_THUMBNAIL": "thumbnail",
```

Comment:
Should this be enough ?

Reply:
Can you "lift" the four ASSET_TYPE_* constants from App.ts into here so they can be used in the backend too? And yeah, that should suffice.

Reply:
This is being handled in [PR 13157](https://github.com/oppia/oppia/pull/13159). After merging of [PR 13157](https://github.com/oppia/oppia/pull/13159) -- I will resolve the conflicts and merge this PR.